### PR TITLE
[Merged by Bors] - feat(Analysis/Normed/Field): define `Normed` instance for `SubfieldClass`

### PR DESCRIPTION
**Pull request by jjdishere**
Add a single instance `SubfieldClass.toNormedField` for induced normed field structure on subfields.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Normed/Field/Basic.lean [hunk 1]
```diff
@@ -1095,6 +1096,19 @@ instance toNormedCommRing [NormedCommRing R] [SubringClass S R] (s : S) : Normed
...
 
+namespace SubfieldClass
+
+variable {S F : Type*} [SetLike S F]
+
+/--
+If `s` is a subfield of a normed field `F`, then `s` is equipped with an induced normed
+field structure.
+-/
+instance toNormedField [NormedField F] [SubfieldClass S F] (s : S) : NormedField s :=
```
Comment (eric-wieser) [inline Mathlib/Analysis/Normed/Field/Basic.lean:1107]:
```suggestion
instance instNormedField [NormedField F] [SubfieldClass S F] (s : S) : NormedField s :=
```
the `to` names are usually reserved for `Foo S -> Bar S`, but this is `[Foo S] (s : S) : Bar s` which is different.

Comment (eric-wieser) [reply Mathlib/Analysis/Normed/Field/Basic.lean:1107]: Ah, I guess this matches the above though!

**Comment by eric-wieser** [comment]
Let's bench this just in case, otherwise LGTM.

**Comment by jjdishere** [comment]
bench!

**Comment by eric-wieser** [comment]
bors d+

**Comment by jjdishere** [comment]
!bench

**Pull request closed without merging**